.drone/config.star

```diff
@@ -0,0 +1,171 @@
+DRONE_BASE_PATH = ".drone/pipelines/default"
+
+def main(ctx):
+    pipeline = {
+        "kind": "pipeline",
+        "type": "docker",
+        "name": "default",
+    }
+
+    pipeline["trigger"] = {
+        "ref": [
+            "refs/heads/test/**",
+            "refs/heads/master",
+        ],
+    }
+
+    steps = []
+
+    steps.append(step(
+        name = "external-binaries",
+        commands = ["bash {}/runtime/binaries.sh".format(DRONE_BASE_PATH)],
```

Comment:
This bash is unnecessary... I will make the script executable and remove the explicit `bash` call. This script is also sourced later by other scripts.

Reply:
Thanks for getting rid of the explicit bash.

I'm thinking of making it be something like `commands = ["@@/runtime/binaries.sh"]`, and have `step()` take care of subbing `@@` (or even just `{}`) with `DRONE_BASE_PATH` (because we essentially do it everywhere and makes things harder to read).  Does that seem like a reasonable idea?

Reply:
I don't think it's the correct approach given that a step doesn't necessarily call a script under the Drone directory. An example is each linting step. We could eventually just call bazel for some of them.

Reply:
I guess I'll live with this for now…
I'm not saying we should prefix _all_ commands; just run a replace on it. But that might end up more confusing than helpful, so ¯\\\_(ツ)\_/¯.